spec/requests/transformation_mappings_spec.rb

```diff
@@ -131,6 +131,34 @@
         expect(response.parsed_body).to include(expected)
         expect(response).to have_http_status(:bad_request)
       end
+
```

Comment:
Hi @AparnaKarve can you add a test that checks that deletes without the appropriate role fails with the forbidden error ?  Thanks.

Reply:
@abellotti I just added a test for that.
Please review. Thanks.